Client/Frontend/Browser/BackForwardListViewController.swift

```diff
@@ -183,14 +196,14 @@ class BackForwardListViewController: UIViewController,
     }
 
     func remakeVerticalConstraints() {
-        guard let bvc = self.bvc else { return }
+        guard let browserFrameInfo = browserViewControllerFrameInfoProvider else { return }
         for constraint in self.verticalConstraints {
             constraint.isActive = false
         }
         self.verticalConstraints = []
         if snappedToBottom {
-            let keyboardContainerHeight = bvc.overKeyboardContainer.frame.height
-            let toolbarContainerheight = bvc.bottomContainer.frame.height
+            let keyboardContainerHeight = browserFrameInfo.getOverKeyoboardContainerSize().height // bvc.overKeyboardContainer.frame.height
+            let toolbarContainerheight = browserFrameInfo.getBottomContainerSize().height // bvc.bottomContainer.frame.height
```

Comment:
We can probably remove the comments
```suggestion
            let keyboardContainerHeight = browserFrameInfo.getOverKeyoboardContainerSize().height
            let toolbarContainerheight = browserFrameInfo.getBottomContainerSize().height
```